tests/e2e/system_test/test_reclaim_space_cronjob.py

```diff
@@ -0,0 +1,212 @@
+import logging
+import pytest
+import yaml
+from uuid import uuid4
+import os
+import time
+from tempfile import NamedTemporaryFile
+
+
+from ocs_ci.framework.testlib import (
+    skipif_ocp_version,
+    skipif_ocs_version,
+    ManageTest,
+    tier3,
+)
+from ocs_ci.utility.utils import run_cmd
+from ocs_ci.helpers.performance_lib import run_oc_command
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.exceptions import PVCNotCreated
+
+logger = logging.getLogger(__name__)
+ERRMSG = "Error in command"
+
+
+@tier3
+@skipif_ocp_version("<4.14")
+@skipif_ocs_version("<4.14")
+class TestReclaimSpaceCronJob(ManageTest):
+    """
+    Test that verifies creation of Reclaim Space Cron Jobs for RBD PCSs in openshift-* namespaces
+    The test also verifies that no Reclaim Space Cron Job is created for CephFS PVC in this namespace
+    """
+
+    def setup(self):
+        self.pvc_objs_created = []
+        self.temp_files_list = []
+
+    @pytest.fixture(autouse=True)
+    def teardown(self, request):
+        """
+        Cleanup the test environment
+
+        """
+
+        def finalizer():
+            # pvc objects usually are deleted in the test (and the second delete has no effect),
+            # if for some reason the pvcs were not deleted in the test - they will be deleted here
+            for pvc_obj in self.pvc_objs_created:
+                pvc_obj.delete()
+
+            for temp_file in self.temp_files_list:
+                if os.path.exists(temp_file):
+                    run_cmd(f"rm {temp_file}")
+
+            run_oc_command(cmd=f"delete namespace {self.namespace}")
+
+        request.addfinalizer(finalizer)
+
+    def test_reclaim_space_cronjob(self):
+        """
+        Test case to check reclaim space cronjobs are created correctly for rbd pvcs in openshift-* namespace
+        """
+        timeout = 30
+        num_of_samples = 10
+        namespace = f"openshift-{uuid4().hex}"
+        self.namespace = namespace
+        res = run_oc_command(cmd=f"create namespace {self.namespace}")
```

Comment:
Yes, unfortunately it is impossible to use OCP. When I tried to use helpers.create_project(namespace) which is successfully used in other tests, I've got the following error: 

21:56:00 - MainThread - ocs_ci.utility.utils - INFO  - Executing command: oc new-project openshift-9ba89cf473414f40b5ec3a6aa1dbd38e
21:56:01 - MainThread - ocs_ci.utility.utils - WARNING  - Command stderr: Error from server (Forbidden): project.project.openshift.io "openshift-9ba89cf473414f40b5ec3a6aa1dbd38e" is forbidden: cannot request a project starting with "openshift-"


Cannot create new project
FAILED

Since it is essential requirement for this test that the namespace will start with "openshift-", the less elegant way of running  oc commands is the one which works